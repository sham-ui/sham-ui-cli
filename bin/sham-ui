#!/usr/bin/env node

require( 'commander' )
    .version( require( '../package' ).version )
    .usage( '<command> [options]' )
    .command( 'new', 'generate a new project from a template' ).alias( 'n' )
    .command( 'generate', 'generate a new component' ).alias( 'g' )
    .parse( process.argv );