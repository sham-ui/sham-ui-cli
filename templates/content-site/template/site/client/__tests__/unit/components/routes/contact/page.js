import { createDI } from 'sham-ui';
import ContactPage  from '../../../../../src/components/routes/contact/page.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const DI = createDI();
    DI.bind( 'title', {
        change() {}
    } );
    const meta = renderer( ContactPage, {
        DI
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
