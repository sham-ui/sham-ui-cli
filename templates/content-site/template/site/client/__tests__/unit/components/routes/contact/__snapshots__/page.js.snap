// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
    "pageTitle": "Contact",
  },
  "html": "
<div data-test-page-body=\\"\\">
  <section class=\\"section pt-55\\">
    <div class=\\"container\\">
      <div class=\\"row\\">
        <div class=\\"container-fluid\\">
          <div class=\\"row\\">
            <div class=\\"categorie-title\\">
              <h3>Contact us</h3>
            </div>
          </div>
        </div>
      </div>
      <div class=\\"row\\">
        <div class=\\"col-lg-10 offset-lg-1 mb-20\\"> For contact with us send email to
          <img src=\\"/images/email.png\\" alt=\\"contact-email\\">
        </div>
      </div>
    </div>
  </section>
  <!--0-->
</div>
<!--0-->
<!--0-->
",
}
`;
